libs/langchain/langchain/vectorstores/redis/base.py

```diff
@@ -132,84 +144,135 @@ class Redis(VectorStore):
             )
     """
 
+    DEFAULT_VECTOR_SCHEMA = {
+        "name": "content_vector",
+        "algorithm": "FLAT",
+        "dims": 1536,
+        "distance_metric": "COSINE",
+        "datatype": "FLOAT32",
+    }
+
     def __init__(
         self,
         redis_url: str,
         index_name: str,
         embedding_function: Callable,
```

Comment:
since we're already making breaking changes, i'd suggest we update this to be
```suggestion
        embedding: Embeddings,
```
which is interface all newer VectorStores have